org.openhab.binding.zigbee/README.md

```diff
@@ -10,6 +10,23 @@ A ZigBee Coordinator is the network controller, and is therefore the heart of th
 
 Coordinators need to be installed manually and the serial port and baud rate must be set. These are set to match the configuration that the dongle is in. Should you wish to use a different baud rate than the default speed of the device, you must change the configuration of the dongle using some other, and then configure the binding to match your change. If in doubt, you should leave the settings at their default values which should work in most cases.
 
+#### Coordinator Configuration
+
+##### Link Key (zigbee_linkkey)
+
+The key is defined as 16 hexadecimal values. If not defined, this will default to the well known ZigBee HA link key.
+
+If defined with the work ```INSTALLCODE:``` before the key, then this will create a link key from an install code which may be shorter than 16 bytes.
```

Comment:
Could these be typos?
* `with the work` -> `with the word`
* `before the key, then this will` -> `before the key, this will`